tests/test_target.py

```diff
@@ -34,11 +47,13 @@ def test_read_multiline_values(self):
         tempdir = tempfile.mkdtemp(prefix='devlib-test-')
```

Comment:
should use context manager classes available in `tempfile`

Reply:
I don't have a change here in this PR.
https://github.com/ARM-software/devlib/pull/667/commits/79a91b7b5f9c35c9329c508de28cb34e4e9365c2 uses context manager anyway.